text/0010-mnemonic-seed/0010-mnemonic-seed.md

```diff
@@ -0,0 +1,59 @@
+- Compute the [SHA-256](https://nvlpubs.nist.gov/nistpubs/FIPS/NIST.FIPS.180-4.pdf) hash of the resulting bit string and use its first 384/32=12 bits as the checksum.
+- The 12-bit checksum is appended to the end of the initial result, making it a 396-bit string.
+- These concatenated bits are split into 36 groups of 11 bits, each encoding a number from 0-2047, corresponding to an index into the wordlist.
+- Finally, convert these numbers into words of any one of the [BIP-0039 wordlists](https://github.com/bitcoin/bips/blob/master/bip-0039/bip-0039-wordlists.md) and use the joined words as a mnemonic sentence.
+
+### Generating the seed from mnemonic
+
+- Convert the 36-word mnemonic sentence into its corresponding 396-bit string, by taking the 11-bit wordlist index for each word and concatenating all the bits.
+- Split the resulting bit string into 384-bit entropy and 12-bit checksum.
+- Verify that the checksum corresponds to the first bits of the SHA-256 hash of the entropy.
+- Convert the 384-bit entropy, interpreted as a signed integer in big-endian two's complement representation, back to a little-endian 243-trit balanced ternary integer. (The most significant trit will always be zero.) This corresponds to the usual 243-trit or 81-tryte IOTA seed.
+
+## Examples
+
+- Using the [English word list](https://github.com/bitcoin/bips/blob/master/bip-0039/english.txt):
+  - IOTA seed (81-tryte): `TLEV9HDTGZOXIIGA9DZG9VAKAIUZKNIMAFUGGARTWPOGDLLVUFZZVAABXRMFPWJAYWBBHOERV9EZBAOJD`
+  - mnemonic (36-word): `forget small borrow baby wing law monkey fiber jealous canyon melt all order lift now fish mind index neither discover divert fit curtain raw wealth arrow frozen plug catalog public winner emerge pulse mixture cry arch`
+- Using the [Japanese word list](https://github.com/bitcoin/bips/blob/master/bip-0039/japanese.txt):
+  - IOTA seed (81-tryte): `KMQDKKLGGTPUBRJXYWLMQOIA9WIEWUAAJPASYPVAWOTYYH9JESDKPLVZIWITHDIUMLFEWQUQ9LHAV9GHC`
+  - mnemonic (36-word): `げどく　まもる　してい　ていへん　つめたい　ちつじょ　だいたい　てうち　まいにち　さゆう　よそく　がはく　ねらう　いちおう　くみあわせ　ふいうち　せつでん　きせい　すべて　きひん　しかい　さぎょう　うけたまわる　つとめる　おんしゃ　きかい　なやむ　たいせつ　うんこう　むすめ　いってい　ふめつ　そとづら　つくね　おいこす　ききて`
+
+# Drawbacks
+
+- This RFC describes a way to represent computer-generated randomness in a human-readable transcription. It is in no way meant to process user created sentences into a binary key. This technique is also sometimes called a "brain wallet" and must not be confused with these mnemonics. 
+- The mnemonics only encode 384 bits of entropy which only covers 242 trits. The 243rd trit will *not* be encoded and always padded with 0. This is perfectly fine, when Kerl is used to derive the private keys, since the Kerl hash function only works on the first 242 trits itself. However, other - currently not used - key derivation functions relaying on the full 243-trit entropy are *not* compatible with this RFC.
```

Comment:
```suggestion
- The mnemonics only encode 384 bits of entropy which only covers 242 trits. The 243rd trit will *not* be encoded and always padded with 0. This is perfectly fine, when Kerl is used to derive the private keys, since the Kerl hash function only works on the first 242 trits itself. However, other - currently not used - key derivation functions relying on the full 243-trit entropy are *not* compatible with this RFC.
```